tests/utils.ts

```diff
@@ -8,9 +9,37 @@ import { fileURLToPath } from 'node:url';
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
 const blueprintPath = path.join(__dirname, '..');
+const fixturesPath = path.join(__dirname, 'fixtures');
 
 export const SUPPORTED_PACKAGE_MANAGERS = ['npm', 'yarn', 'pnpm'] as const;
 
+export async function fixture(
```

Comment:
I was not sure what this is doing when seeing it used in code, like generating a fixture, asserting against one, or reading it (which is happening here). Maybe rename to `readFixture`?